lithops/serverless/backends/aws_lambda/config.py

```diff
@@ -32,7 +32,8 @@
     '3.6': 'python3.6',
     '3.7': 'python3.7' ,
     '3.8': 'python3.8' ,
-    '3.9': 'python3.9'
+    '3.9': 'python3.9' ,
+    '3.10': 'python3.10'
```

Comment:
This is only for the default runtime, and the default runtime does not support python 3.10 yet

Reply:
Okay!